src/attr.rs

```diff
@@ -0,0 +1,94 @@
+#[repr(C)]
```

Comment:
Maybe rename this to src/property_attribute ?

Reply:
Done!